src/views/ChallengeDetail.vue

```diff
@@ -98,35 +98,37 @@ export default {
         }
       })
     },
-    getStats() {
+    async getStats() {
       this.loading = true
-      api.getPriceStats({ ...this.defaultParams, product__categories_tags__contains: this.challenge.categories[0] })
-      .then((data) => {
-        this.challenge.numberOfContributions = data.price__count
-        this.loading = false
-      })
+      let priceCount = 0
+      for (let i = 0; i < this.challenge.categories.length; i++) {
+        const data = await api.getPriceStats({ ...this.defaultParams, product__categories_tags__contains: this.challenge.categories[i] })
+        priceCount += data.price__count
+      }
+      this.challenge.numberOfContributions = priceCount
 
-      api.getProofs({ ...this.defaultParams, size: 1 })
-      .then((data) => {
-        this.challenge.numberOfProofs = data.total
-      })
+      const proofsStats = await api.getProofs({ ...this.defaultParams, size: 1 })
+      this.challenge.numberOfProofs = proofsStats.total
 
       if (this.username) {
-        api.getPriceStats({ ...this.defaultParams, product__categories_tags__contains: this.challenge.categories[0], owner: this.username })
-        .then((data) => {
-          this.challenge.userContributions = data.price__count
-        })
-        api.getProofs({ ...this.defaultParams, owner: this.username, size: 1 })
-        .then((data) => {
-          this.challenge.userProofContributions = data.total
-        })
+        let userPriceCount = 0
+        for (let i = 0; i < this.challenge.categories.length; i++) {
```

Comment:
ok I'd rather have something cleaner ^^ (without the loop, without the async)
I'm going to have a look at the backend and unlock the multiple-category filtering :)